cdc/changefeed.go

```diff
@@ -847,6 +895,103 @@ func (c *changeFeed) pullDDLJob() error {
 	return nil
 }
 
+// startSyncPeriod start a timer for every changefeed to create sync point by time
+func (c *changeFeed) startSyncPeriod(ctx context.Context, interval time.Duration) {
+	log.Debug("sync ticker start", zap.Duration("sync-interval", interval))
+	go func(ctx context.Context) {
+		ticker := time.NewTicker(interval)
+		for {
+			select {
+			case <-ctx.Done():
+				return
+			case <-ticker.C:
+				c.updateResolvedTs = false
+			}
+		}
+	}(ctx)
+}
+
+//createSynctable create a sync table to record the
+func (c *changeFeed) createSynctable(ctx context.Context) error {
+	database := "TiCDC"
+	tx, err := c.syncDB.BeginTx(ctx, nil)
+	if err != nil {
+		log.Info("create sync table: begin Tx fail")
+		return err
```

Comment:
```suggestion
		return  cerror.WrapError(cerror.ErrMySQLTxnError, err)
```
We use a new RFC error mechanism now, the error from third library should be wrapped by an RFC error generator. Ditto for some other errors. Note if the error is generated from RFC error, wrap is not needed